modules/raft/src/main/java/org/apache/ignite/internal/raft/server/impl/RaftGroupEventsListenerAdapter.java

```diff
@@ -62,10 +62,24 @@ class RaftGroupEventsListenerAdapter implements JraftGroupEventsListener {
     }
 
     @Override
-    public void onLeaderElected(long term) {
+    public void onLeaderElected(
+            long term,
+            long configurationTerm,
+            long configurationIndex,
+            Collection<PeerId> peers,
+            Collection<PeerId> learners
+    ) {
         serviceEventInterceptor.onLeaderElected(grpId, term);
 
-        delegate.onLeaderElected(term);
+        delegate.onLeaderElected(
+                term,
+                configurationTerm,
+                configurationIndex,
+                PeersAndLearners.fromConsistentIds(
```

Comment:
you can use `org.apache.ignite.internal.raft.server.impl.RaftGroupEventsListenerAdapter#configuration` here